### PR TITLE
Use logger

### PR DESCRIPTION
**Pull request by anhmuz**

### CODE REVIEW

File: KokrjachApi/KokrjachApi/KokrjachApi.csproj.user [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <ActiveDebugProfile>Docker</ActiveDebugProfile>
+    <ActiveDebugProfile>KokrjachApi</ActiveDebugProfile>
```
Comment (hulakov) [inline KokrjachApi/KokrjachApi/KokrjachApi.csproj.user]: Is it really needed?

File: KokrjachApi/KokrjachApi/Controllers/EventsController.cs [hunk 2]
```diff
@@ -42,7 +42,7 @@ public EventsController(ILogger<EventsController> logger)
         [HttpGet]
         public IEnumerable<Event> GetEvents()
         {
-            Console.WriteLine("Get a list of events");
+            _logger.LogInformation("Get a list of events");
```
Comment (hulakov) [inline KokrjachApi/KokrjachApi/Controllers/EventsController.cs:45]: Have you configured output to stdout?

Comment (anhmuz) [reply KokrjachApi/KokrjachApi/Controllers/EventsController.cs:45]: Yes

File: KokrjachApi/KokrjachApi/KokrjachApi.csproj.user (hunk 1 above)
Comment (anhmuz) [reply KokrjachApi/KokrjachApi/KokrjachApi.csproj.user]: No. Removed

**Review by hulakov — APPROVED** [review]

**Pull request merged**